Separate channel for odoo instances

roman-marchenko-advisability opened the pull request.

roman-marchenko-advisability committed:
Separate channel for odoo instances

dbadvisability commented on queue_job/jobrunner/channels.py:
```diff
@@ -6,6 +6,7 @@
 from heapq import heappop, heappush
 from weakref import WeakValueDictionary
 
+from . import queue_job_config
```
може тут абсолютный путь сделать?

roman-marchenko-advisability replied:
скажем так тут вообще нехорошая штука получается
при старте еще вся ода не стартанута(соответственно аддоны не собраны, registry не собран), а job_queue в server_wide_modules и получаем ошибку ModuleNotFoundError, но старт оды идет дальше

да и вообще queue_job_config в  __init__.py завернут в try/except и там линтер уже будет ругаться

dbadvisability replied:
а, точно. из-за server_wide_modules получается только с точкой

roman-marchenko-advisability committed:
Refactor channel notification logic in jobrunner

Simplify and enhance the `notify` method by adjusting the logic for handling the `job_server_only` flag. Introduced a fallback mechanism for non-root channels when the flag is not set, improving flexibility and behavior consistency.
